editor/editor_node.h

```diff
@@ -682,6 +682,7 @@ class EditorNode : public Node {
 
 protected:
 	friend class FileSystemDock;
+	friend class DebugAdapterParser;
```

Comment:
This should be removed, see the review comment.

Reply:
Fixed :+1: